lib/faker/util.ex

```diff
@@ -273,5 +273,22 @@ defmodule Faker.Util do
     end
   end
 
+  @doc """
+  Removes both single and double quotes from the given string.
+
+  ## Examples
+
+      iex> Faker.Util.remove_special_characters("f'oo")
+      "foo"
+      iex> Faker.Util.remove_special_characters(~s(f"o'o))
+      "foo"
+  """
+  @spec remove_special_characters(binary) :: binary
+  def remove_special_characters(string) do
```

Comment:
I don't think we should move it out, the name is too generic, we won't be able to reuse it if other characters in other places would be different. Lets keep it in `lib/faker/internet.ex` for now.

Reply:
I wasn't able to test it in isolation because it was a private function, which is why I pulled it out. Would you be open to renaming it to `remove_quote_characters` or something like that?